operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -199,6 +257,21 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
                 LOG.finest("The request specified a '" + START_QUERY_PARAMETER + "' and/or '" + END_QUERY_PARAMETER + "' query parameter. They are not valid for resource type '" + compartmentType + "', so will be ignored.");
                 searchParameters = queryParametersWithoutDates;
             }
+            
+            // Need to get the value of the _type parameter out, if it exists
+            List<String> typeParameters = new ArrayList<String>(0);
+            List<Parameters.Parameter> parameterList = parameters.getParameter();
+            for (Parameters.Parameter parameter: parameterList) {
+                typeParameters = new ArrayList<String>(parameterList.size());
+                if (parameter.getName().getValue().equals("_type")) {
+                    String[] types = parameter.getValue().as(ModelSupport.FHIR_STRING).getValue().split(",");
+                    Collections.addAll(typeParameters, types);
+                    break;
```

Comment:
what about _type=Condition&_type=CarePlan - the cardinality is 

```
_type | 0..* -- | --
```

I'd expect it to loop through all

Reply:
This code was removed.

Reply:
In a subsequent commit?

Reply:
Yes, it is gone in the commit I applied last night.